LightBlue.MultiHost/Runners/NpmRunner.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
```

Comment:
It's interesting that `NpmRunner`, `DotNetCoreRunner` and `ConsoleRunner` only differ by a few lines.  Feels like the actual runner is a `ShellRunner` or `ExternalProcess` runner, and the npm\dotnetcore\console aspects are just injected path & args (to internally pass down to ProcessStartInfo)

Reply:
Currently the axis of change is reflected in the name of the runner (.NET core, NPM, Console).  To me this seems like a good application of SRP that would support changes along those axis.  The ShellRunner would couple those three responsibilities in favour of DRY, is that better?